src/plugins/intel_gna/src/frontend/model_quantizer.hpp

```diff
@@ -147,8 +147,8 @@ class ModelQuantizer {
             if (inf_loop_history.empty()) {
                 inf_loop_count++;
             } else {
-                if (inf_loop_count > 0 &&
-                    (inf_loop_history.size() % inf_loop_pattern.size() == 0 || sf.allLayersProcessed()) &&
+                if ((inf_loop_count > 0) &&
+                    ((inf_loop_history.size() % inf_loop_pattern.size() == 0) || sf.allLayersProcessed()) &&
```

Comment:
Why is this change present here? Is it related to this PR in some way?

Reply:
removed